### PR TITLE
Finalize diagnostics for preprint/submission

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

For this PR:
- Removed trailing white space and double spaces
- Checked acronyms were defined at first mention & used at least once after definition
- Ran through Grammarly's plagiarism checker (reported 100% unique once I excluded matches to this repository)
- Tried to use Grammarly to review the sentence clarity etc but it was SO slow, so their changes were only made to the abstract (I used to copy edit, so hopefully I'll do a decent job on my own)
- Checked scite badges for references (only concerning one was the ML study that we addressed in #1132)
- Final read through in docx output for sentence clarity, grammar, etc.
- Check/remove all comments from md

If it looks alright, I'll go ahead and start the preprint submission!

### Related issues

#1094 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/10.diagnostics.md [hunk 1]
```diff
@@ -131,11 +130,12 @@ First, the authors evaluated samples from the 103 suspected cases.
...
+
 They repeated this analysis in patient samples from contacts and close contacts.
 Patients who tested negative with both methods (n = 48) were observed to remain healthy over a period of 14 days.
-Among the remaining 29 samples from contacts, RT-qPCR identified 12 as positive, 1 as negative, and 16 as borderline. 
+Among the remaining 29 samples from contacts, RT-qPCR identified 12 as positive, 1 as negative, and 16 as borderline.
 All of the samples that tested positive using RT-qPCR also tested positive using ddPCR.
-In contrast, the negative result and all but one of the borderline results were identified as positive by RT-ddPCR, and these patients were later determined to be SARS-CoV-2 positive based on clinical evaluation and repeated molecular sampling. <!--It is not entirely clear to me from the methods & supplement whether this resampling was done with RT-qPCR (section 3.5 in the reference). However, as noted above, they do specify the technique used for suspected cases (right about Fig 3 in the paper).-->
```
Comment (rando2) [inline content/10.diagnostics.md:138]: Due to the lack of info in this paper, it seems inadvisable to clarify this statement, so I'm removing the comments.

File: content/10.diagnostics.md [hunk 2]
```diff
@@ -213,7 +213,7 @@ It also reported 100% specificity and sensitivity on 114 RNA samples from clinic
 A separate study screened four designs of SHERLOCK and extensively tested the best-performing assay.
 They determined the limit of detection to be 10 copies/&mu;l using both fluorescent and lateral flow detection [@doi:10.1101/2020.02.26.967026].
 
-LFT strips are simple to use and read, but there are limitations in terms of availability and cost per test. <!-- Is SHERLOCK more available and less expensive than LFTs? This sentence feels out of place, or I'm missing the point. -->
```
Comment (rando2) [inline content/10.diagnostics.md:216]: This sentence ended up in the wrong paragraph when a paragraph break was added. I think it's a lot more obvious now what it is supposed to mean (but let me know if that's incorrect!)

File: content/10.diagnostics.md [hunk 3]
```diff
@@ -244,28 +245,27 @@ They rely on type V (Cas12-based) and type IV (Cas13-based) CRISPR systems.
...
 Lateral flow strips showed 100% correlation to the high-throughput DETECTR assay.
 Importantly, DETECTR was 100% specific for SARS-CoV-2 and did not detect other human coronaviruses.
-A method based on a Cas9 ortholog from _Francisella novicida_ (FnCas9) achieved 100% sensitivity and 97% specificity in clinical samples, and the diagnostic kit is reported to have completed regulatory validation in India [@doi:10.1101/2020.09.13.20193581].
+A method based on a Cas9 ortholog from _Francisella novicida_ known as FnCas9 achieved 100% sensitivity and 97% specificity in clinical samples, and the diagnostic kit is reported to have completed regulatory validation in India [@doi:10.1101/2020.09.13.20193581].
 
 #### Immunoassays for the Detection of Antigens
 
 Immunoassays can detect molecular indicators of SARS-CoV-2 infection, such as the proteins that act as antigens from the SARS-CoV-2 virus.
-<!-- Move this: Since 2020, many immunoassays have been developed for COVID-19 [@url:https://www.centerforhealthsecurity.org/covid-19TestingToolkit/serology/Serology-based-tests-for-COVID-19.html] using several different technologies.-->
```
Comment (rando2) [inline content/10.diagnostics.md:268]: This is a note I made to myself during revision that I don't think we need to listen to

File: content/10.diagnostics.md [hunk 4]
```diff
@@ -292,14 +292,13 @@ Despite having lower sensitivity than PCR tests, LFTs occupy an important niche
 Current infection detection by LFTs enables the scale and speed of testing that is beneficial to managing viral spread.
 LFTs were available freely to citizens in the United Kingdom until April 1, 2022 [@url:https://www.gov.uk/government/news/government-sets-out-next-steps-for-living-with-covid] and to citizens of the United States in early 2022 [@url:https://www.whitehouse.gov/briefing-room/statements-releases/2022/01/14/fact-sheet-the-biden-administration-to-begin-distributing-at-home-rapid-covid-19-tests-to-americans-for-free].
 These tests are particularly useful for ruling out SARS-CoV-2 infection in cases where the likelihood of infection is low (e.g., asymptomatic individuals) and positives (including false positives) can be validated with testing by alternate means [@doi:10/hk34].
-<!--BinaxNOW LFI very low sensitivity [@doi:10.3201/eid2710.210080].-->
```
Comment (rando2) [inline content/10.diagnostics.md:295]: Another note I made for myself

**Review by agitter — APPROVED** [review]
Looks good to me, as do your explanations of the deleted comments.

**Comment by agitter** [comment]
I updated the dictionary in [198664e](https://github.com/greenelab/covid19-review/pull/1137/commits/198664e200cbef873cd09cfaaa50ab1018312e66) to deal with the figure reference.

**Comment by RLordan** [comment]
Thanks @rando2

**Pull request merged**